crates/turborepo-lib/src/config/client.rs

```diff
@@ -17,12 +19,22 @@ struct ClientConfigValue {
 #[derive(Debug, Clone)]
 pub struct ClientConfigLoader {
     remote_cache_timeout: Option<u64>,
+    environment: Option<HashMap<String, String>>,
 }
 
 impl ClientConfig {
     #[allow(dead_code)]
```

Comment:
Y'know I'm not entirely sure why we have these everywhere in config.

Reply:
It's an interesting _bonus_ method of setting these things. It makes the most sense for concurrent testing strategies so I actually think it's worth keeping.